daemon/container.h

```diff
@@ -395,6 +395,18 @@ container_has_setup_mode(const container_t *container);
 bool
 container_contains_pid(const container_t *container, pid_t pid);
 
+int
```

Comment:
remove this API, this should not be necessary anymore since this is handles internally in c_seccomp

Reply:
done